src/models/db/transfers.js

```diff
@@ -7,8 +9,32 @@ function * getTransfer (id, options) {
   return (yield Transfer.findById(id, options))
 }
 
+function * _upsertTransfer (transfer, options) {
+  assert(options.transaction)
+  // SQLite's implementation of upsert does not tell you whether it created the
```

Comment:
I think this comment is a little dated. Since we are not actually calling `upsert` on a db model, maybe just say that we need the correct status code (200/201), so we need to know if the transfer exists in the db.